admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/domain/entity/dukan/Dukan.kt

```diff
@@ -1,11 +1,28 @@
 package net.thechance.mena.admin_panel.domain.entity.dukan
 
+import kotlin.uuid.ExperimentalUuidApi
+import kotlin.uuid.Uuid
+
+@OptIn(ExperimentalUuidApi::class)
 data class Dukan(
-    val id: String,
+    val id: Uuid,
     val name: String,
     val imageUrl: String,
     val address: String,
     val latitude: Double,
     val longitude: Double,
-    val categories: List<Category>
-)
\ No newline at end of file
+    val categories: List<Category>,
+    val activationStatus: ActivationStatus,
+    val status: Status
+) {
+    enum class ActivationStatus {
+        ACTIVATED,
+        DEACTIVATED,
+    }
```

Comment:
Here you can add valueOfOrDefault function like I mentioned above, same for Status enum 
```suggestion
    enum class ActivationStatus {
        ACTIVATED,
        DEACTIVATED;
        
            companion object {
            fun valueOfOrDefault(value: String?): Dukan.ActivationStatus {
                return runCatching { value?.let { Dukan.ActivationStatus.valueOf(it) } ?: ACTIVATED }.getOrDefault(
                    ACTIVATED
                )
            }
        }
     }
```